Sources/TestingMacros/PragmaMacro.swift

```diff
@@ -27,6 +29,17 @@ public struct PragmaMacro: PeerMacro, Sendable {
     providingPeersOf declaration: some DeclSyntaxProtocol,
     in context: some MacroExpansionContext
   ) throws -> [DeclSyntax] {
+    if case let .argumentList(arguments) = node.arguments,
+       arguments.first?.label?.textWithoutBackticks == "warning" {
+      let targetNode = Syntax(declaration)
+      let messages = arguments
```

Comment:
Surprised to not see `.lazy` in this chain :)

Reply:
We don't use `.lazy` as much in the macro target, you know that! :P

Reply:
Hmm, seems just as useful there as elsewhere to me

Reply:
Sure. Not something we need to fix here, but I don't personally see a reason to _avoid_ our general coding style of using `.lazy` in the macro implementation.

Reply:
Our general style has been _not_ to use it in the macro implementation. :)